lab1/exam.py

```diff
@@ -0,0 +1,7 @@
+ninjas = int(input("введите количество врагов:"))
+if (ninjas <= 10):
+  print("Я одолею этих ниндзя!")
+elif (ninjas >= 30):
+  print("Их слишком много")
+elif (ninjas < 30 or ninjas >= 10):
+  print("Будет непросто, но я с ними разделаюсь")
```

Comment:
Стоит обернуть в функцию - а код вызова перенести в if __name...
```suggestion
exam(ninjas):  
  if (ninjas <= 10):
    print("Я одолею этих ниндзя!")
  elif (ninjas >= 30):
    print("Их слишком много")
  elif (ninjas < 30 or ninjas >= 10):
    print("Будет непросто, но я с ними разделаюсь")
  if __name__ == "__main__":  
    ninjas = int(input("введите количество врагов:"))
    exam(5)
```